tensorboard/lazy_test.py

```diff
@@ -77,6 +78,17 @@ def foo():
     foo.namedtuple
     self.assertEquals(repr(foo), "<%r via LazyModule (loaded)>" % collections)
 
+  def test_failed_load_idempotent(self):
+    @lazy.lazy_load("tensorboard.lazy_test_bad_import")
+    def bad():
+      from tensorboard import lazy_test_bad_import
```

Comment:
This file isn't present in your PR.  Do we actually need a separate file for this?  Can't we just define bad to `raise Exception`?

Reply:
Oops, thanks! Will change to `raise Exception`.